Bottom: 41e62895531a21d352988e232606f6e598ca9374
Top:    7ebd19c0850d9914df38fbf61fb8b81eeb23e0e5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-11-13 00:19:11 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
